weavel/types/websocket.py

```diff
@@ -1,7 +1,7 @@
 from enum import StrEnum
-from typing import Any, Dict, Iterable, List, TypedDict, Union
+from typing import Any, Dict, Iterable, List, Optional, TypedDict, Union
 from openai.types.chat.completion_create_params import ChatCompletionMessageParam
```

Comment:
The previous code was also wrong, but `TypedDict` should always be imported from `typing_extensions` since `from typing import TypedDict` is only supported in python >= 3.12